contracts/src/v0.8/operatorforwarder/test/operator.t.sol

```diff
@@ -0,0 +1,311 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.19;
+
+import {Test} from "forge-std/Test.sol";
+import {Operator} from "../Operator.sol";
+import {ChainlinkClientHelper} from "./testhelpers/ChainlinkClientHelper.sol";
+import {LinkToken} from "../../shared/token/ERC677/LinkToken.sol";
+
+import "./testhelpers/Deployer.t.sol";
+import "../AuthorizedReceiver.sol";
+
+contract OperatorTest is Deployer, AuthorizedReceiver {
+  address public s_link;
+  uint256 private dataReceived;
+  ChainlinkClientHelper public s_client;
+  Operator public s_operator;
+
+  function setUp() public {
+    _setUp();
+    s_link = address(new LinkToken());
+    s_client = new ChainlinkClientHelper(s_link);
+
+    address[] memory auth = new address[](1);
+    auth[0] = address(this);
+    s_operator = new Operator(s_link, address(this));
+    s_operator.setAuthorizedSenders(auth);
+
+    dataReceived = 0;
+  }
+
+  // Callback function for oracle request fulfillment
+  function callback(bytes32 _requestId) public {
```

Comment:
I prefer using a helper contract over using the test files themselves as receivers for callbacks